README.md

```diff
@@ -3,7 +3,14 @@
 Converts text to Spongebob chicken meme mocking case. On your discord server, invoke using `!spongify Python is a great scripting language`
 
 ## Getting Started
-
+```bash
+pip install -r requirements-dev.txt
+# create a git pre-commit hook for linting
```

Comment:
From here to line 13 is only needed if you developing. May want to give it its own header